docs/modules/ROOT/pages/index.adoc

```diff
@@ -45,6 +45,8 @@ To keep your system secure, you should **always** use the installed code as-is,
 
 Please report any security issues you find via our https://www.immunefi.com/bounty/openzeppelin[bug bounty program on Immunefi] or directly to security@openzeppelin.org.
 
+The https://contracts.openzeppelin.com/security[OpenZeppelin Contracts Security Center] contains more details about the secure development process.
```

Comment:
I checked in the deployment preview and the name sounds a bit repetitive in the site, how do we feel about replacing by the following:

```suggestion
The https://contracts.openzeppelin.com/security[Security Center] contains more details about the secure development process.
```

Also, have we consulted with marketing if we should use the full "Open Zeppelin"?